Cython/Compiler/Tests/TestCmdLine.py

```diff
@@ -514,3 +528,9 @@ def error(*args):
         error('--verbose=1')
         error('--cleanup')
         error('--debug-disposal-code-wrong-name', 'file3.pyx')
+        # No source file (source file appears to be module name).
+        error('--module-name', 'foo.pyx')
+        # Cannot use --module-name with more than one source file.
+        error('--module-name', 'foo.bar', 'foo.pyx', 'bar.pyx')
+        # Cannot use --module-name with --timestamps
+        error('--module-name', 'foo.bar', '--timestamps', 'foo.pyx')
```

Comment:
Would be nice to start (regex) testing for error messages here. Better than adding them in comments.

Reply:
I will have a go.